drivers/gpu/drm/rp1/rp1-vec/rp1_vec.c

```diff
@@ -362,11 +368,13 @@ static int rp1vec_connector_get_modes(struct drm_connector *connector)
 			} else if (mode->hdisplay == 704 && mode->vtotal == preferred_lines) {
 				mode->type |= DRM_MODE_TYPE_PREFERRED;
 			}
+			mode->picture_aspect_ratio = par;
 			drm_mode_set_name(mode);
+
 			drm_mode_probed_add(connector, mode);
 			n++;
 
-			if (mode->vtotal == 405 || mode->vtotal == 819)
+			if (i >= 4)
```

Comment:
This 4 is creeping in as a magic number both here and line 355. It'd be nice if it came from a define in and around the rp1vec_modes array.